packages/grid/_modules_/grid/models/colDef/gridColDef.ts

```diff
@@ -71,6 +71,10 @@ export interface GridColDef {
    * @default 'string'
    */
   type?: GridColType;
+  /**
+   * To be used in combination with `type='select'`. Array with possible cell values.
```

Comment:
This is not a React prop.

```suggestion
   * To be used in combination with `type: 'select'`. This is an array of the possible cell values and labels.
```